src/harness/testcases/WINNF_FT_S_PPR_testcase.py

```diff
@@ -401,3 +409,257 @@ def test_WINNF_FT_S_PPR_2(self, config_filename):
+    device_b['installationParam']['latitude'] = 38.815291
+    device_b['installationParam']['longitude'] = -97.198805
+    device_c = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_c.json')))
+    device_c['installationParam']['latitude'] = 38.816782
+    device_c['installationParam']['longitude'] = -97.102965
+    device_d = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_d.json')))
+    device_d['installationParam']['latitude'] = 38.758462
+    device_d['installationParam']['longitude'] = -97.189036
+    device_e = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_e.json')))
+    device_e['installationParam']['latitude'] = 38.761748
+    device_e['installationParam']['longitude'] = -97.118459
+
+    # Pre-load conditionals and remove REG-conditional fields from registration
+    # requests.
+    conditional_keys = [
+        'cbsdCategory', 'fccId', 'cbsdSerialNumber', 'airInterface',
+        'installationParam', 'measCapability'
+    ]
+    reg_conditional_keys = [
+        'cbsdCategory', 'airInterface', 'installationParam', 'measCapability'
+    ]
+    conditionals_b = {key: device_b[key] for key in conditional_keys}
+    device_b = {
+        key: device_b[key]
+        for key in device_b
+        if key not in reg_conditional_keys
+    }
+    conditionals_d = {key: device_d[key] for key in conditional_keys}
+    device_d = {
+        key: device_d[key]
+        for key in device_d
+        if key not in reg_conditional_keys
+    }
+
+    # Load grant requests (default is 3620-3630).
+    grant_a = json.load(
+        open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    grant_b = json.load(
+        open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    grant_c = json.load(
+        open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    grant_d = json.load(
+        open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    grant_d['operationParam']['operationFrequencyRange'][
+        'lowFrequency'] = 3620000000
+    grant_d['operationParam']['operationFrequencyRange'][
+        'highFrequency'] = 3640000000
+    grant_e = json.load(
+        open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+
+    # CBSDs in SAS UUT.
+    domain_proxy = {
+        'registrationRequests': [device_b, device_d],
+        'grantRequests': [grant_b, grant_d],
+        'conditionalRegistrationData': [conditionals_b, conditionals_d],
+        'cert': getCertFilename('domain_proxy.cert'),
+        'key': getCertFilename('domain_proxy.key')
+    }
+
+    # One PPA in SAS UUT.
+    pal_low_frequency = 3620000000
+    pal_high_frequency = 3630000000
+    pal_record_0 = json.load(
+        open(os.path.join('testcases', 'testdata', 'pal_record_0.json')))
+    ppa_record_0 = json.load(
+        open(os.path.join('testcases', 'testdata', 'ppa_record_0.json')))
+    ppa_record_0, pal_records_0 = makePpaAndPalRecordsConsistent(
```

Comment:
The fips code entry is missing for this record (pal_record_0)

Reply:
It's included (see [here](https://github.com/Wireless-Innovation-Forum/Spectrum-Access-System/blob/master/src/harness/testcases/testdata/pal_record_0.json)), I just didn't need to change it from the default so I didn't set it here.

Reply:
Yup see it now. Thanks